script/deployment/StoryNFT.s.sol

```diff
@@ -0,0 +1,69 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.26;
+
+import { Base64 } from "@openzeppelin/contracts/utils/Base64.sol";
+import { ILicenseRegistry } from "@storyprotocol/core/interfaces/registries/ILicenseRegistry.sol";
+
+import { IStoryBadgeNFT } from "../../contracts/interfaces/story-nft/IStoryBadgeNFT.sol";
+import { IStoryNFT } from "../../contracts/interfaces/story-nft/IStoryNFT.sol";
+import { DeployHelper } from "../utils/DeployHelper.sol";
+
+contract StoryNFT is DeployHelper {
+    address internal CREATE3_DEPLOYER = 0x384a891dFDE8180b054f04D66379f16B7a678Ad6;
+    uint256 private constant CREATE3_DEFAULT_SEED = 4987979255314221321141221123132325342;
+    constructor() DeployHelper(CREATE3_DEPLOYER) {}
+
+    function run() public override {
+        create3SaltSeed = CREATE3_DEFAULT_SEED;
+        writeDeploys = true;
+
+        _readStoryProtocolCoreAddresses();
+        (address defaultLicenseTemplate, uint256 defaultLicenseTermsId) =
+            ILicenseRegistry(licenseRegistryAddr).getDefaultLicenseTerms();
+        address storyNftFactorySigner = vm.envAddress("STORY_NFT_FACTORY_SIGNER");
+        address rootOrgNftRecipient = vm.envAddress("ROOT_ORG_NFT_RECIPIENT");
+        address rootStoryNftOwner = vm.envAddress("ROOT_STORY_NFT_OWNER");
+        address rootStoryNftSigner = vm.envAddress("ROOT_STORY_NFT_SIGNER");
+        string memory rootOrgName = "Test Root Org";
+        string memory rootOrgTokenURI = string(abi.encodePacked(
```

Comment:
What's the purpose of the script? we should not deploy such Test Org in product env, sounds right?

Reply:
Yeah production shouldn't use these values. I used this script to deploy test contracts.

Reply:
In that case, the file name should be renamed to `StoryNFTTest.s.sol` for better readability.

I didn't find a script to only for deploying StoryNFT related contracts, should we add such script?

Reply:
I can take out the root org deployment logic and handle that with a local script. Then, we’ll just use `StoryNFTTest.s.sol` to deploy `StoryNFTFactory` and `OrgNFT`.

Reply:
I would suggest to remove all test logic and keep StoryNFT.s.sol as deployment script to deploy official StoryNFT Factory and template contracts.

Reply:
sounds good! will make the changes :)

Reply:
Resolved in [c644e86](https://github.com/storyprotocol/protocol-periphery-v1/pull/87/commits/c644e8656d81fa0b132d160d6ea5aacad3dcaf4a) :)